Compiling and linking: ../bin/bsdtar

Dear all,
I am trying to compile the suite on Ubuntu 20.04 LTS.

After a fresh start with all default folder names, I `cmake-gui ../openchemistry` and I configure the Makefile as following:

![image](https://user-images.githubusercontent.com/11757284/117647122-df033800-b18c-11eb-9b2d-c2c955a17621.png)

I then I configure, generate and close. Afterwards I run a `cmake --build .`, which compiles almost till the end. The error reported is:

```
Scanning dependencies of target bsdtar
[ 94%] Building C object tar/CMakeFiles/bsdtar.dir/bsdtar.c.o
[ 94%] Building C object tar/CMakeFiles/bsdtar.dir/cmdline.c.o
[ 94%] Building C object tar/CMakeFiles/bsdtar.dir/creation_set.c.o
[ 95%] Building C object tar/CMakeFiles/bsdtar.dir/read.c.o
[ 95%] Building C object tar/CMakeFiles/bsdtar.dir/subst.c.o
[ 96%] Building C object tar/CMakeFiles/bsdtar.dir/util.c.o
[ 96%] Building C object tar/CMakeFiles/bsdtar.dir/write.c.o
[ 96%] Building C object tar/CMakeFiles/bsdtar.dir/__/libarchive_fe/err.c.o
[ 97%] Building C object tar/CMakeFiles/bsdtar.dir/__/libarchive_fe/line_reader.c.o
[ 97%] Building C object tar/CMakeFiles/bsdtar.dir/__/libarchive_fe/passphrase.c.o
[ 98%] Linking C executable ../bin/bsdtar
/usr/bin/ld: ../libarchive/libarchive.a(archive_read_support_format_rar5.c.o): in function `reset_file_context':
/home/giammi56/Documents/openchemistry-build/thirdparty/libarchive-src/libarchive/archive_read_support_format_rar5.c:853: undefined reference to `blake2sp_init'
/usr/bin/ld: ../libarchive/libarchive.a(archive_read_support_format_rar5.c.o): in function `update_crc':
/home/giammi56/Documents/openchemistry-build/thirdparty/libarchive-src/libarchive/archive_read_support_format_rar5.c:2353: undefined reference to `blake2sp_update'
/usr/bin/ld: ../libarchive/libarchive.a(archive_read_support_format_rar5.c.o): in function `verify_checksums':
/home/giammi56/Documents/openchemistry-build/thirdparty/libarchive-src/libarchive/archive_read_support_format_rar5.c:3870: undefined reference to `blake2sp_final'
collect2: error: ld returned 1 exit status
make[5]: *** [tar/CMakeFiles/bsdtar.dir/build.make:226: bin/bsdtar] Error 1
make[4]: *** [CMakeFiles/Makefile2:1104: tar/CMakeFiles/bsdtar.dir/all] Error 2
make[3]: *** [Makefile:141: all] Error 2
make[2]: *** [thirdparty/CMakeFiles/libarchive.dir/build.make:115: thirdparty/libarchive-prefix/src/libarchive-stamp/libarchive-build] Error 2
make[1]: *** [CMakeFiles/Makefile2:390: thirdparty/CMakeFiles/libarchive.dir/all] Error 2
make: *** [Makefile:84: all] Error 2
```
I have checked around and `bsdtar` has been moved to `libarchive-tools` [after Ubuntu 19.04 ](https://github.com/intel/lkp-tests/issues/50). The compiled `libarchive.a` and .so live in `/usr/lib/x86_64-linux-gnu`. I have tried to update `libarchive-tools` and `libarchive-tools-dev` and checked the flag `USE_SYSTEM_LIBARCHIVE`, but the configuration cmake repots that:

```
CMake Error at thirdparty/CMakeLists.txt:18 (find_package):
  By not providing "FindLIBARCHIVE.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "LIBARCHIVE", but CMake did not find one.

  Could not find a package configuration file provided by "LIBARCHIVE" with
  any of the following names:

    LIBARCHIVEConfig.cmake
    libarchive-config.cmake

  Add the installation prefix of "LIBARCHIVE" to CMAKE_PREFIX_PATH or set
  "LIBARCHIVE_DIR" to a directory containing one of the above files.  If
  "LIBARCHIVE" provides a separate development package or SDK, be sure it has
  been installed.
```

Is there any possible solution?

Thank you!

--- a/cmake/projects.cmake
+++ b/cmake/projects.cmake
@@ -69,9 +69,9 @@ set(spglib_md5 "1933e2252a0e708951ee476eb93f6495")
 
 # libarchive
 list(APPEND projects libarchive)
-set(libarchive_version "3.5.1")
+set(libarchive_version "3.3.3")
 set(libarchive_url "https://github.com/libarchive/libarchive/archive/v${libarchive_version}.tar.gz")
-set(libarchive_md5 "742a47936955f280b9dc712ac544f8ec")
+set(libarchive_md5 "c591a2781609e7d0c47049a0f061ce99")
 
 # msgpackc
 list(APPEND projects msgpackc)

